identity_cache.gemspec

```diff
@@ -38,20 +38,8 @@ Gem::Specification.new do |gem|
   gem.add_dependency('ar_transaction_changes', '~> 1.1')
   gem.add_dependency('activerecord', '>= 5.2')
 
-  gem.add_development_dependency('memcached', '~> 1.8.0')
-  gem.add_development_dependency('memcached_store', '~> 1.0.0')
-  gem.add_development_dependency('dalli')
-  gem.add_development_dependency('rake')
-  gem.add_development_dependency('mocha', '0.14.0')
-  gem.add_development_dependency('spy')
-  gem.add_development_dependency('minitest', '>= 2.11.0')
-
-  if RUBY_PLATFORM == 'java'
```

Comment:
I believe this changes behaviour when someone tries to run this gem on JRuby, right?

Reply:
It was just raising before.  However, there is no need to add that constraint here, since Identity Cache doesn't itself require MRI ruby, even though we end up testing with MRI specific backends.